lattice-controller/README.md

```diff
@@ -1,7 +1,21 @@
 # Lattice Controller Capability Provider
 
-A capability provider that allows actors to interact with the lattice control interface via
-the NATS substrate.
+A capability provider that allows actors to interact with the lattice control interface (`wasmcloud:latticecontrol`) by
+remotely communicating with lattices and the hosts contained within them via NATS.
+
+## Configuration
+This capability provider is designed to facilitate connections to multiple lattices. For each connection that actors need to utilize, there must be an accompanying `set_lattice_credentials` call, _even if you are establishing a connection to the `default` lattice_.
```

Comment:
We don't do this in all of our provider READMEs, but since https://crates.io/crates/wasmcloud-interface-lattice-control is no longer compatible with this provider I would love to see just a sample actor function that does something with this provider. This could be at the bottom of the README:
```rust
async fn start_actor(ctx: &Context) -> RpcResult<CtlOperationAck> {
    let lattice = LatticeControllerSender::new();
    // TODO: set lattice creds
    let cmd = StartActorCommand {
        lattice_id: "default".to_string(),
        actor_ref: "wasmcloud.azurecr.io/echo:0.3.4".to_string(),
        annotations: None,
        count: 250,
        host_id: "NB67YNOVU5YB3526RUNCKNZBCQDH2L5NZJKQ6FWOVWGSHNHHEO65RP4A".to_string(),
    };

    debug!(
        "Starting {} instance(s) of actor {} on host {}",
        cmd.count, cmd.actor_ref, cmd.host_id
    );

    lattice.start_actor(ctx, &cmd).await
}
```